style.css

```diff
@@ -455,102 +457,329 @@ input {
 }
 
 /* Media queris */
-@media only screen and (max-width: 600px) {
-  a {
-    text-decoration: none;
-    color: #091e42;
+@media only screen and (min-width: 768px) {
+  .row {
+    flex-direction: row;
   }
 
-  .about-content h1 {
-    margin: 10px;
+  .col-1 img {
+    width: 100%;
+  }
+
+  .big-button-2:hover {
+    background-color: #6070ff;
+    color: white;
+  }
+
+  .about-myself h1 {
     color: #091e42;
-    font-weight: 800;
-    font-size: 2rem;
-    width: 60%;
+    font-size: 40px;
+    line-height: 52px;
+    text-align: left;
+    vertical-align: center;
+    width: 20%;
+    margin: 10px;
   }
+}
 
-  .about-content p {
+/* Humburger menu */
+.nav-menu {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 60px;
+}
+
+.navbar {
+  min-height: 70px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 24px;
+}
+
+.nav-branding {
+  font-size: 1.5rem;
+}
+
+.nav-link {
+  transition: 0.8s ease;
+}
+
+.hamburger-btn {
+  display: none;
+}
+
+.bar {
+  display: block;
+  margin: 5px auto;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: ll 0.3s ease-in-out;
+  background-color: #6070ff;
+  width: 30px;
+  height: 4px;
+}
+
+@media (max-width: 768px) {
+  .hamburger-btn {
+    display: block;
+  }
+
+  .hamburger-btn.active .bar:nth-child(2) {
+    opacity: 0;
+  }
+
+  .hamburger-btn.active .bar:nth-child(1) {
+    transform: translateY(8px) rotate(45deg);
+  }
+
+  .hamburger-btn.active .bar:nth-child(3) {
+    transform: translateY(-8px) rotate(-45deg);
+  }
+
+  .nav-menu {
+    height: 100vh;
+    position: fixed;
+    top: 0;
+    left: -120%;
+    gap: 5px;
+    justify-content: center;
+    padding: 10% 5% 50% 2%;
+    width: 100vw;
+    transition: 0.7s ease-out;
+    flex-direction: column;
+    background: #fff;
+  }
+
+  .nav-list .nav-link {
     color: #344563;
-    width: 60%;
-    margin: 10px;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 20px;
   }
 
-  .about-content h4 {
-    color: #7f8cff;
-    line-height: 24px;
-    font-size: 16px;
-    margin: 10px 10px 10px 0;
-    margin: 10px;
+  .nav-menu .nav-list {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    align-items: center;
+    height: 70px;
+    color: #396df2;
   }
 
-  .about-content ul {
-    padding: 10px;
-    color: #505f79;
-    width: 60%;
+  .nav-menu.active {
+    left: 0;
+  }
+
+  .Popup-section {
+    position: fixed;
+    top: 0;
+    left: 0;
+    min-height: 100vh;
+    background: rgba(0, 0, 0, 0.8);
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    display: none;
+  }
+
+  .Popup-section .Popup-container {
+    margin: 2% 5% 2% 4%;
+    background: #fff;
+    border-radius: 20px;
+    position: relative;
+    max-height: 100%;
+    display: none;
+  }
```

Comment:
- [ ] KIndly make sure your pop-up is well-centered following the width and height of the page as is the case on the Figma template.
- [ ] Also make sure you use the same color as the Figma design for the `blur`. 
Ypur design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/197226564-e57845a5-b109-4734-b007-6fa2d574a0f6.png)

Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/197226832-21495b27-4f7b-4072-86a9-917193d7ec40.png)

Reply:
@christianbiring1  I did not get that how ?